### PR TITLE
[Term entry] Go-math-function-min

### PR DESCRIPTION
**Pull request by Manikanta528**
### Description

Addressing issue #2559 

### Type of Change

- Adding a new entry go math function min

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/go/concepts/math-functions/terms/min/min.md
Comment (caupolicandiaz) [inline content/go/concepts/math-functions/terms/min/min.md:28]: I noticed that these examples are the same as in the max entry. It makes sense to want to simplify and use what you've created before, but preferably all examples are original.

**Review by caupolicandiaz — APPROVED** [review]
@Manikanta528 Overall, looks good. Just left one comment. This is ready for a second review.

**Comment by SSwiniarski** [comment]
@Manikanta528 Looks good to merge once you address @caupolicandiaz's comment.

File: content/go/concepts/math-functions/terms/min/min.md
Comment (Manikanta528) [reply content/go/concepts/math-functions/terms/min/min.md:28]: What should i do?

Comment (Manikanta528) [reply content/go/concepts/math-functions/terms/min/min.md:28]: Should I change the examples?

Comment (SSwiniarski) [reply content/go/concepts/math-functions/terms/min/min.md:28]: @Manikanta528 Yes, go ahead and change them. Then tag us so we can merge this.

Comment (Manikanta528) [reply content/go/concepts/math-functions/terms/min/min.md:28]: @SSwiniarski yes, updated the examples.

**Review by SSwiniarski — APPROVED** [review]

**Comment by SSwiniarski** [comment]
@Manikanta528 Congrates on the new entry! Live link here: https://www.codecademy.com/resources/docs/go/math-functions/min

image

**Pull request merged**
